tap/tlstapper/tls_poller.go

```diff
@@ -0,0 +1,83 @@
+package tlstapper
+
+import (
+	"bufio"
+	"bytes"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/up9inc/mizu/shared/logger"
+	"github.com/up9inc/mizu/tap/api"
+)
+
+const UNKWONW_PORT string = "80"
+const UNKWONW_HOST string = "127.0.0.1"
+
+func Poll(tls *TlsTapper, httpExtension *api.Extension,
+	emitter api.Emitter, options *api.TrafficFilteringOptions) {
+
+	chunks := make(chan *tlsChunk)
+
+	go tls.pollPerf(chunks)
+
+	for {
+		chunk := <-chunks
+
+		ip, port, err := chunk.getAddress()
+
+		if err != nil {
+			logger.Log.Warningf("Error getting address from tls chunk %v", err)
+			continue
+		}
+
+		var id api.TcpID
+
+		reader := bufio.NewReader(bytes.NewReader(chunk.Data[0:chunk.Recorded]))
+		isRequest := (chunk.isClient() && chunk.isWrite()) || (chunk.isServer() && chunk.isRead())
+		
+		if isRequest {
+			id = api.TcpID{
+				SrcIP:   UNKWONW_HOST,
+				DstIP:   ip.String(),
+				SrcPort: UNKWONW_PORT,
+				DstPort: strconv.FormatInt(int64(port), 10),
+			}
+		} else {
+			id = api.TcpID{
+				SrcIP:   ip.String(),
+				DstIP:   UNKWONW_HOST,
+				SrcPort: strconv.FormatInt(int64(port), 10),
+				DstPort: UNKWONW_PORT,
+			}
+		}
+
+		httpExtension.Dissector.Dissect(reader, isRequest, &id, &api.CounterPair{}, &api.SuperTimer{}, &api.SuperIdentifier{}, emitter, options)
+
+		if os.Getenv("MIZU_VERBOSE_TLS_TAPPER") == "true" {
```

Comment:
Why introducing a new environment variable for debug logs? Why can't we just use the global debug level?

Reply:
It's too much

Reply:
Too verbose OK understandable.

Reply:
Tapper code very spammy, and that makes debug prints hard to use.
I think that granular control over prints would be helpful.